easybuild/easyconfigs/c/Clang/Clang-15.0.5-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,51 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2013-2015 Dmitri Gribenko, Ward Poelmans
+# Authors:: Dmitri Gribenko <gribozavr@gmail.com>
+# Authors:: Ward Poelmans <wpoely86@gmail.com>
+# License:: GPLv2 or later, MIT, three-clause BSD.
+# $Id$
+##
+
+name = 'Clang'
+version = '15.0.5'
+
+homepage = 'https://clang.llvm.org/'
+description = """C, C++, Objective-C compiler, based on LLVM.  Does not
+ include C++ standard library -- use libstdc++ from GCC."""
+
+# Clang also depends on libstdc++ during runtime, but this dependency is
+# already specified as the toolchain.
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ["https://github.com/llvm/llvm-project/releases/download/llvmorg-%(version)s"]
+sources = [
+    'llvm-project-%(version)s.src.tar.xz',
+]
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+    ('Perl', '5.34.1'),
+    ('elfutils', '0.187'),
+]
+dependencies = [
+    # since Clang is a compiler, binutils is a runtime dependency too
+    ('binutils', '2.38'),
+    ('hwloc', '2.7.1'),
+    ('libxml2', '2.9.13'),
+    ('ncurses', '6.3'),
+    ('GMP', '6.2.1'),
+    ('Z3', '4.10.2'),
+    ('Python', '3.10.4'),
```

Comment:
`libclang` has python bindings, but I guess this is really a very optional dependency. I leave it in for now.

Reply:
We already have `Clang-Python-bindings` for the those binding. So I suggest to keep `Clang` without the Python bindings.

Reply:
Removed them and left a comment in place